appinventor/components/src/com/google/appinventor/components/runtime/ButtonBase.java

```diff
@@ -373,8 +374,15 @@ private void updateAppearance() {
         } else {
           // Clear the background image.
           ViewUtil.setBackgroundDrawable(view, null);
-          // Set to the specified color (possibly COLOR_NONE for transparent).
-          TextViewUtil.setBackgroundColor(view, backgroundColor);
+          //Now we set again the default drawable
```

Comment:
@NmdOfficial Should line 375 (which sets the BackgroundDrawable to null) be deleted, or does it serve a purpose (which I am not clear on)?

Reply:
No the line should not be deleted.